pkg/cli/gptscript.go

```diff
@@ -359,23 +371,7 @@ func (r *GPTScript) Run(cmd *cobra.Command, args []string) (retErr error) {
 				gptOpt.Env = append(gptOpt.Env, "SCRIPTS_PATH="+cwd)
 			}
 
-			if os.Getenv(system.BinEnvVar) == "" {
-				gptOpt.Env = append(gptOpt.Env, system.BinEnvVar+"="+system.Bin())
-			}
-
-			// Pass the corrected environment variables for SDK server options
-			if r.DefaultModel != "" {
-				gptOpt.Env = append(gptOpt.Env, "GPTSCRIPT_SDKSERVER_DEFAULT_MODEL="+r.DefaultModel)
-			}
-			if len(r.CredentialOverride) > 0 {
-				gptOpt.Env = append(gptOpt.Env, "GPTSCRIPT_SDKSERVER_CREDENTIAL_OVERRIDE="+strings.Join(r.CredentialOverride, ","))
-			}
-
 			gptOpt.Env = append(gptOpt.Env, "UI_RUN_FILE="+file)
-
-			if len(args) > 2 {
-				args = append(args, args[2:]...)
-			}
```

Comment:
Is this piece no longer needed? Just want to make sure its removal is not accidental.

Reply:
I did intend to remove it. The original intention of this code was to remove the file from the args and keep the remaining args. I have added logic to this PR for the same purpose.